templates/ra_guac_autoscale_public_alb.template.cfn.yaml

```diff
@@ -89,6 +93,18 @@ Parameters:
     Default: ''
     Description: Text/Label to display branding for the Guac Login page
     Type: String
+  CloudWatchAgentUrl:
+    AllowedPattern: '^$|^s3://.*\.rpm$'
+    Default: ''
+    Description: >-
+      (Optional) S3 URL to CloudWatch Agent installer. Example(CentOS):
+      s3://amazoncloudwatch-agent/amazon_linux/amd64/latest/amazon-cloudwatch-agent.rpm
+    Type: String
+  CloudWatchAppLogs:
```

Comment:
This template is _specific_ to the guac app, so we don't really need to expose a parameter for the logs we want the app to send, we can just include them in the template.